tiny_dnn/util/macro.h

```diff
@@ -12,3 +12,9 @@
 #if defined _WIN32 && !defined(__MINGW32__)
 #define CNN_WINDOWS
 #endif
+
+#ifdef _MSC_VER
+#define CNN_BE_INLINE_ALREADY __forceinline
+#else
+#define CNN_BE_INLINE_ALREADY __attribute__((always_inline)) inline
+#endif
```

Comment:
nyanp suggested to add a fallback to inline so I think it'd be good to follow it.
https://github.com/tiny-dnn/tiny-dnn/pull/733#discussion_r118842307

And I recommended to use the macro name `CNN_BE_INLINE_ALREADY` as a joke. :joy: